content/docs/guide-development.md

```diff
@@ -24,13 +24,26 @@ You can use a mock [lightcurve notebook](https://renkulab.io/gitlab/astronomy/mm
 ### Parametetrize the notebook 
 
 * create a cell with the following tag "parameters" (see [papermill manual](https://papermill.readthedocs.io/en/latest/usage-parameterize.html#designate-parameters-for-a-cell)):
+  * if not annotated, the types of the inputs parameters are determined based on the parameter default value
+  * one can annotate the input parameter by putting comment with the `term` from the [ontology](https://odahub.io/docs/guide-ontology).
+  * several parameters are mandatory in the MMODA platform. These includes:
```

Comment:
could you add a more clear note on what "mandated" implies here? I guess we can assume the workflow will not be allowed to deploy if these are missing.

Reply:
> Yes, these parameters are mandated by the frontend. But “mandatory” is probably the wrong word here; currently, if the nb doesn't use these parameters, it's ok, the plugin just adds them for compatibility.

Ah ok, then let's explain this. They are not mandatory, but recommended common parameters.

> The downside, is that exactly the same result could be produced with different parameter values, which is not ideal at all.

Unfortunately, this already happens in plenty of cases. E.g. ANTARES time dimension (but also in other cases). Maybe we can always add a warning like with ANTARES?

> Also, the graph produced by renku plugin will not correspond to the mmoda service.

It will have the relevant information. We can make the necessary inference after.

> 
> Maybe, we should introduce changes to the dispatcher to allow requests with these parameters for the workflows, that don't use them?

Not sure what you mean?

Reply:
> Not sure what you mean?

I recall that the only problem we get, if these parameters are missing in the instrument, but one sends request with them (e.g. by api code from the frontend), then oda_api will return warnings. The easiest step is just to suppress such warnings, everything will continue to work. Or change the warning to something more human-friendly, like the note about times in Antares frontend.   
A more complex solution is to filter unneeded parameters from the request (and so from the generated API code, product urls etc) at the dispatcher level. 

Such a way we don't need to mandate these parameters in any sense

Reply:
ok, lets do here https://github.com/oda-hub/hugo-odahub/issues/14